tests/ui/panicking_macros.rs

```diff
@@ -33,9 +35,18 @@ fn unreachable() {
     let b = a + 2;
 }
 
+fn core_versions() {
+    use core::{panic, todo, unimplemented, unreachable};
+    panic!();
+    todo!();
+    unimplemented!();
+    unreachable!();
```

Comment:
I think all of them should emit a warning. It shouldn't matter where those macros are coming from.

Reply:
Yes, that's what this PR is trying to do.

Reply:
> Ugh, now only `core::unreachable` is warning. Ideas?

Oh I missed the "only" in this sentence, nvm.